src/resources/DatabaseCancelDeployment.yaml

```diff
@@ -0,0 +1,23 @@
+post:
```

Comment:
Should we not use a DELETE @erebe ?

Reply:
Post or Delete is fine to me (as we don't really delete but cancel it)

Reply:
Ok looks good to me